cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+select = B,B9,C,D,DAR,E,F,N,RST,S,W
+ignore = E203,E501,RST201,RST203,RST301,W503,D100
+max-line-length = 120
```

Comment:
think, with cookietemple we were at 160, you want to reduce it?

Reply:
Yes. I personally like 160, but so many people think that it is too much. They're stuck with small laptops and want to code with 2 windows open at the same time.

Reply:
Ok, accepted ;)